packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -369,6 +370,28 @@ describe('Right Click Menu integration', () => {
     );
   }
 
+  /**
+   * Add product to canvas
+   *
+   * @param productPartial Object with product properties to override defaults.
```

Comment:
```suggestion
   * @param {Object} productPartial Object with product properties to override defaults.
```

To avoid the JSDoc lint warning